django/river/common/analyzer/cleaning_script.py

```diff
@@ -1,26 +1,20 @@
 import logging
 
-import scripts
+from common.scripts import Script
 
 logger = logging.getLogger(__name__)
 
 
 class CleaningScript:
```

Comment:
I'm considering whether we really need this wrapper around `Script` or not... What if `apply` and `eq` were defined in `Script`?

Reply:
`apply` would not have the same implem for for classes, but I definitely can use inheritance.